src/lib/components/radio/props.ts

```diff
@@ -5,7 +5,7 @@ import type { NativeProps } from "$lib/utils";
 export interface QRadioProps extends NativeProps, HTMLAttributes<HTMLLabelElement> {
   value: string;
   label?: string;
-  selected?: any;
+  selected?: unknown;
```

Comment:
What's this prop for, and why is it unknow at compile time?

Reply:
Short story: Because I couldn't find the proper type. But it should be addressed in the future.